lambdas/models/bulk_upload_report.py

```diff
@@ -28,6 +29,11 @@ class BulkUploadReport(BaseModel):
     uploader_ods_code: str = Field(alias=MetadataReport.UploaderOdsCode)
     reason: Optional[str] = Field(default="", alias=MetadataReport.Reason)
 
+    def get_registered_at_uploader_practice_status(self):
```

Comment:
Sorry forgot to add this as part of the last review, could we also add a return type for this function as it will be `-> str` and then cast the response of the boolean `self.uploader_ods_code == self.pds_ods_code` to return as a string value also? Just so it's clear whats being done here as we've returning two different data types from a glance. somethging like this?
```
def get_registered_at_uploader_practice_status(self) -> str:
    if self.pds_ods_code in PatientOdsInactiveStatus.list():
        return self.pds_ods_code
    return str(self.uploader_ods_code == self.pds_ods_code)
```